Dapper.SqlBuilder/SqlBuilder.cs

```diff
@@ -149,5 +149,8 @@ protected SqlBuilder AddClause(string name, string sql, object parameters, strin
 
         public SqlBuilder Having(string sql, dynamic parameters = null) =>
             AddClause("having", sql, parameters, "\nAND ", "HAVING ", "\n", false);
+
+        public SqlBuilder Set(string sql, dynamic parameters = null) =>
+             AddClause("set", sql, parameters, " , ", "Set ", "\n", false);

```

Comment:
```suggestion
             AddClause("set", sql, parameters, " , ", "SET", "\n", false);

```
I'm all for title case, but let's be consistent here :)